### ISSUE TITLE
Default Value

### ISSUE THREAD

**Issue by betocantu93** [tracked-tools/tracked-toolbox#7]
Passing fallback value errors out 🤔 

```ts
@localCopy('args.showCreateDialog', false) showCreateDialog;
```

```console
You attempted to update `__tag_value__` on `Tag`, but it had already been used previously in the same computation.  Attempting to update a value after using it in a computation can cause logical errors, infinite revalidation bugs, and performance issues, and is not supported.
```

**Comment by jordpo** [comment]
What version of Ember are you running on? I don't see this issue on 3.20.

**Comment by betocantu93** [comment]
@jordpo 3.16 last time I checked

**Comment by chriskrycho** [comment]
Per comment on #17, and without further repro, I'm going to close this as it seems to have been fixed in the improvements landed in Ember between 3.16 and 3.20 (which were substantial, as were the refactors up through 3.28, especially around 3.23 and 3.25). Thanks!
